cognite/replicator/datapoints.py

```diff
@@ -142,14 +142,13 @@ def replicate_datapoints_several_ts(
                                     f"Could not manipulate the datapoint (value={src_datapoint.value},"
                                     + f" timestamp={src_datapoint.timestamp}). Error: {e}"
                                 )
-                if transformed_dps is not None:
-                    list_of_datapoints = transformed_dps
-                else:
-                    list_of_datapoints = [
-                        {"timestamp": datapoints.timestamp[i], "value": datapoints.value[i]}
-                        for i in range(len(datapoints.timestamp))
-                    ]
-                logging.info(f"Ext id:  {datapoints.external_id} Number of datapoints: {len(list_of_datapoints)}")
+            if transformed_dps is not None:
```

Comment:
What is the reason for unindenting this part of code ? Win't that be an issue if it is moved outside of the for loop ?